tests/validator/test_validator.py

```diff
@@ -1007,45 +1007,53 @@ def test_validator_include_rendered_content(
         )
     )
 
-    expected_expectation_validation_result_rendered_content = RenderedAtomicContent(
-        name="atomic.diagnostic.observed_value",
-        value=RenderedAtomicValue(
-            schema={"type": "com.superconductive.rendered.string"},
-            params={},
-            template="6",
-        ),
-        value_type="StringValueType",
+    expected_expectation_validation_result_diagnostic_rendered_content = (
+        RenderedAtomicContent(
+            name="atomic.diagnostic.observed_value",
+            value=RenderedAtomicValue(
+                schema={"type": "com.superconductive.rendered.string"},
+                params={},
+                template="6",
+            ),
+            value_type="StringValueType",
+        )
     )
 
     assert (
-        expected_expectation_validation_result_rendered_content
+        expected_expectation_validation_result_diagnostic_rendered_content
         in validation_result.rendered_content
     )
 
-    expected_expectation_configuration_rendered_content = RenderedAtomicContent(
+    expected_expectation_configuration_diagnostic_rendered_content = RenderedAtomicContent(
         name="atomic.prescriptive.summary",
         value=RenderedAtomicValue(
             schema={"type": "com.superconductive.rendered.string"},
             params={
                 "column": {"schema": {"type": "string"}, "value": "passenger_count"},
-                "min_value": {"schema": {"type": "number"}, "value": 1},
-                "max_value": {"schema": {"type": "number"}, "value": 8},
-                "eval_param__0": {
-                    "schema": {"type": "string"},
-                    "value": "min_value: upstream_column_min",
+                "min_value": {
+                    "schema": {"type": "number"},
+                    "value": 1,
+                    "evaluation_parameter": {
+                        "schema": {"type": "object"},
+                        "value": {"$PARAMETER": "upstream_column_min"},
+                    },
                 },
-                "eval_param__1": {
-                    "schema": {"type": "string"},
-                    "value": "max_value: upstream_column_max",
+                "max_value": {
+                    "schema": {"type": "number"},
+                    "value": 8,
+                    "evaluation_parameter": {
+                        "schema": {"type": "object"},
+                        "value": {"$PARAMETER": "upstream_column_max"},
+                    },
                 },
             },
-            template="$column maximum value must be greater than or equal to $min_value and less than or equal to $max_value.   $eval_param__0, $eval_param__1",
+            template="$column maximum value must be greater than or equal to $min_value and less than or equal to $max_value.",
```

Comment:
@josectobar I will hold off on merge until you confirm this works for the diagnostic view.